src/components/Trashy/TrashySchedule.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, FormattedDate } from 'react-intl';
+import ExternalLink from 'components/ExternalLink';
+import I18nLink from 'components/I18n/I18nLink';
+import { curbsideServices as i18n } from 'js/i18n/definitions';
+import { injectIntl } from 'react-intl';
+
+const TrashySchedule = props => {
+  const { address, nextBulkPickupDate, pickupDates, intl } = props;
+
+  const mapService = s => {
+    let service = s.toLowerCase();
+    {
+      var cases = {
+        textiles: intl.formatMessage(i18n.textiles),
+        recycling: intl.formatMessage(i18n.recycling),
+        yardtrimmings: intl.formatMessage(i18n.yardtrimmings),
+        trash: intl.formatMessage(i18n.trash),
+        compost: intl.formatMessage(i18n.compost),
+        brushcollection: intl.formatMessage(i18n.brushcollection),
+        bulkitemcollection: (
+          <I18nLink to="bulk-item-pickup">
+            {intl.formatMessage(i18n.bulkitemcollection)}
+          </I18nLink>
+        ),
+      };
+      return cases[service];
+    }
+  };
+
+  const pickUpList = pickupDates.map(item => {
+    const services = item.services.map(service => {
+      let s = mapService(service);
+      return <li>{s}</li>;
+    });
+
+    return (
+      <div className="coa-Trashy__schedule-list-pickup-item">
+        <h4>
+          <FormattedDate
+            value={item.date}
+            timeZone="UTC"
+            weekday="long"
+            year="numeric"
+            month="long"
+            day="2-digit"
+          />
+        </h4>
+        <ul>{services}</ul>
+      </div>
+    );
+  });
+
+  const bulkPickupDate = (
+    <FormattedDate
+      value={nextBulkPickupDate}
+      timeZone="UTC"
+      weekday="long"
+      year="numeric"
+      month="long"
+      day="2-digit"
+    />
+  );
+
+  const bulkItemPickupLink = (
+    <I18nLink to="bulk-item-pickup">
+      {intl.formatMessage(i18n.bulkitemcollection)}
+    </I18nLink>
+  );
+
+  return (
+    <div className="coa-Trashy__schedule-container">
+      <div className="coa-Trashy__schedule-header">
+        <h3>
+          <FormattedMessage
```

Comment:
Instead of using FormattedMessage, could we use intl.formatMessage where possible, so we can create these translation definitions in src/js/i18n/definitions.js so we have one central place for definitions that can easily be shared between components?